### PR TITLE
Update ready for release `v1.1.0`

### PR DESCRIPTION
**Pull request by krowlandson**
## Overview/Summary

Update docs ready for release `v1.1.0`

## This PR fixes/adds/changes/removes

1. Version updates in documentation
2. Version update in `base_module_tags` local variable
2. Update `customise` (`en-gb`) to `customize` (`en-us`)
3. Remove regionalization from Microsoft Docs links

### Breaking Changes

None

## Testing Evidence

n/a

## As part of this Pull Request I have

- [x] Checked for duplicate [Pull Requests](https://github.com/Azure/terraform-azurerm-caf-enterprise-scale/pulls)
- [x] Associated it with relevant [issues](https://github.com/Azure/terraform-azurerm-caf-enterprise-scale/issues), for tracking and closure.
- [x] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/terraform-azurerm-caf-enterprise-scale/tree/main)
- [x] Performed testing and provided evidence.
- [x] Updated relevant and associated documentation.
- [x] Updated the ["What's New?"](https://github.com/Azure/Enterprise-Scale/wiki/Whats-new) wiki page (located in the [Enterprise-Scale repo](https://github.com/Azure/Enterprise-Scale) in the directory: `/docs/wiki/whats-new.md`)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -208,11 +208,40 @@ For the latest examples, please refer to our [Examples][wiki_examples] guide on
 
 ## Release Notes
 
-Release `v1.0.0` is now here.
-Functionality remains largely similar to `v0.4.0`, but we have made a few minor bug fixes and the following breaking changes should be noted:
-- The minimum supported version of the AzureRM Provider is now set to `2.77.0`
+Release `v1.1.0` introduces the following changes:
+
+- **BREAKING CHANGE**: Replace `Deploy-ASC-Configuration` Policy Assignment with `Deploy-ASCDF-Config`, utilizing built-in policies and adding support for [Microsoft Defender for open-source relational databases](https://docs.microsoft.com/azure/defender-for-cloud/defender-for-databases-introduction).
```
Comment (jtracey93) [inline README.md:213]:
```suggestion
- **BREAKING CHANGE**: Replaced `Deploy-ASC-Configuration` Policy Assignment with `Deploy-ASCDF-Config`, utilizing built-in policies and also adds support for [Microsoft Defender for open-source relational databases](https://docs.microsoft.com/azure/defender-for-cloud/defender-for-databases-introduction).
```

File: README.md [hunk 2]
```diff
@@ -208,11 +208,40 @@ For the latest examples, please refer to our [Examples][wiki_examples] guide on
...
+  - Fixed [Add Defender support for Open-source relational databases #131](https://github.com/Azure/terraform-azurerm-caf-enterprise-scale/issues/131).
```
Comment (jtracey93) [inline README.md:214]:
```suggestion
  - Fixing [Add Defender support for Open-source relational databases #131](https://github.com/Azure/terraform-azurerm-caf-enterprise-scale/issues/131).
```

File: README.md [hunk 3]
```diff
@@ -208,11 +208,40 @@ For the latest examples, please refer to our [Examples][wiki_examples] guide on
...
+  - Will result in loss of policy compliance history.
```
Comment (jtracey93) [inline README.md:215]:
```suggestion
  - **Note:** Will result in loss of policy compliance history.
```

**Review by jtracey93 — CHANGES_REQUESTED** [review]

**Review by matt-FFFFFF — DISMISSED** [review]
LGTM

**Review by jtracey93 — APPROVED** [review]
🚢

**Pull request merged**